package.json

```diff
@@ -5,13 +5,14 @@
     "packages/*"
   ],
   "devDependencies": {
-    "@deck.gl/carto": "^8.7.0",
-    "@deck.gl/core": "^8.7.0",
-    "@deck.gl/extensions": "^8.7.0",
-    "@deck.gl/geo-layers": "^8.7.0",
-    "@deck.gl/google-maps": "^8.7.0",
-    "@deck.gl/layers": "^8.7.0",
-    "@deck.gl/mesh-layers": "^8.7.0",
+    "@deck.gl/aggregation-layers": "^8.7.2",
```

Comment:
I had to add this new dependency for the build to pass. Is it normal?

Reply:
<img width="1203" alt="image" src="https://user-images.githubusercontent.com/9151432/159432277-96341b08-b008-43ed-a103-7c153dd69e16.png">

@deck.gl/aggregation-layers is a new peer-dependencies of @deck.gl/carto

Reply:
Yes, Felix added it after my comment :)
That's why I'm actually waiting for 8.7.3 to be released to update directly to that version.

Reply:
Great 👌🏻

Reply:
@Clebal I've updated to 8.7.3